kin-core/src/test/java/kin/core/KeyStoreImplTest.java

```diff
@@ -33,78 +30,26 @@
     public ExpectedException expectedEx = ExpectedException.none();
     private KeyStoreImpl keyStore;
 
-    private static class FakeEncryptor implements Encryptor {
-
-        @Override
-        public String encrypt(String secret) throws CryptoException {
-            return "{FAKE}" + secret + "{FAKE}";
-        }
-
-        @Override
-        public String decrypt(String encryptedSecret) throws CryptoException {
-            return encryptedSecret.replace("{FAKE}", "");
-        }
-    }
-
-    private static class ExceptionThrowerEncryptor implements Encryptor {
-
-        @Override
-        public String encrypt(String secret) throws CryptoException {
-            throw new CryptoException(new Exception("Some bad things happened"));
-        }
-
-        @Override
-        public String decrypt(String encryptedSecret) throws CryptoException {
-            throw new CryptoException(new Exception("Some bad things happened"));
-        }
-    }
-
-    private static class ExceptionThrowerStore implements Store {
-
-        HashMap<String, String> map = new HashMap<>();
-
-        @Override
-        public void saveString(@NonNull String key, @NonNull String value) {
-            map.put(key, value);
-        }
-
-        @Nullable
-        @Override
-        public String getString(@NonNull String key) {
-            return "NotJsonString";
-        }
-
-        @Override
-        public void clear(@NonNull String key) {
-            map.remove(key);
-        }
-    }
-
     @Before
     public void setup() {
-        keyStore = new KeyStoreImpl(new FakeStore(), new FakeEncryptor());
+        keyStore = new KeyStoreImpl(new FakeStore());
```

Comment:
you can remove this setup and set the store as you need on each test, because not all the test use the same store